src/public/js/search.js

```diff
@@ -63,6 +74,12 @@ $(function () {
         case 'go':
           setSearchOn('EPFL Go', arrayOfTerms[1]);
           break;
+        case 'epnet':
```

Comment:
Can you add a `case 'dir':`?

Reply:
Sure. But please do your comments all at once, it's annoying to change it while re-submitted.